src/Forms/Shared/Samples/GeometryEngine/CreateGeometries/CreateGeometries.xaml.cs

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Xamarin.Forms;
+
+namespace ArcGISRuntime.Samples.CreateGeometries
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Create geometries",
+        "GeometryEngine",
+        "This sample demonstrates simple ways to create different types of Geometry. The geometries are added as Graphics to a GraphicsOverlay and displayed in a MapView. An Envelope is used to set the Viewpoint of the MapView.",
+        "",
+        "")]
```

Comment:
Same comment about tags as above, I think this applies to all of the platforms, so I won't comment again.